eng/Versions.props

```diff
@@ -22,7 +22,7 @@
     <MicrosoftFSharpCompilerPackageVersion>10.4.2-rtm-190219-02</MicrosoftFSharpCompilerPackageVersion>
     <MicrosoftNETCoreCompilersPackageVersion>3.1.0-beta1-19125-02</MicrosoftNETCoreCompilersPackageVersion>
     <MicrosoftNETSdkRazorPackageVersion>3.0.0-preview4.19123.2</MicrosoftNETSdkRazorPackageVersion>
-    <MicrosoftNETSdkWebPackageVersion>3.0.100-preview3.19114.6</MicrosoftNETSdkWebPackageVersion>
+    <MicrosoftNETSdkWebPackageVersion>3.0.100-preview.19124.1</MicrosoftNETSdkWebPackageVersion>
```

Comment:
Looks like a downgrade, but I suspect websdk is producing preview (not preview*4*) branded builds out of its master branch. @vijayrkn is that so?

Reply:
We haven't updated the branding in master. Let me update it to preview4.

Reply:
Thanks.